src/main/java/io/reactivex/rxjavafx/observers/BindingObserver.java

```diff
@@ -66,17 +70,23 @@ public void onError(Throwable e) {
 
     @Override
     public void onNext(T t) {
-        value = t;
-        fireValueChangedEvent();
+        try {
+            value = unmaskingFunction.apply(t);
+            fireValueChangedEvent();
+        } catch (Exception e) {
+            onError(e);
```

Comment:
I'm not sure if passing the exception to onError is the right approach

Reply:
It looks ok to me, ```BindingObserver.onError(throwable)``` apply user error management if provided or the exception is ignored.